crates/stc_ts_file_analyzer/tests/pass/types/literal/stringMappingOverPatternLiterals/2.swc-stderr

```diff
@@ -19,4 +19,4 @@ Error:
    `----
 
 Error: 
-  > `AA${number}${bigint}${boolean}`
+  > `AA${number}${bigint}${(TRUE | FALSE)}`
```

Comment:
So these are generated by running the rust runner I presume but I dont think I understand the output. Is there any way I can confirm whether this is correct or not? :thinking:

Reply:
You can copy the input typescript file to the typescript playground and hover on the type to see the _correct_ type

Reply:
Great, thanks!

Reply:
Initially I just wanted to fix the one error but I should try to do it the right way (similarly how tsc does it) so it has the correct types as well and covers more cases. so I closed it for now

Reply:
I see, thank you!